test/k8sT/Services.go

```diff
@@ -2760,6 +2760,26 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 					testNodePort(true, false, false, 0) // no need to test from outside, as testDSR did it
 				})
 
+				It("Should not drop SCTP packets", func() {
+					vm1 := helpers.InitRuntimeHelper(helpers.K8s1VMName(), logger)
+					vm2 := helpers.InitRuntimeHelper(helpers.K8s2VMName(), logger)
+					ctx, cancel := context.WithCancel(context.Background())
+					defer cancel()
+
+					By("Running iperf3 server")
+					_ = vm1.ExecInBackground(ctx, "iperf3 -s")
+					serverIpAddress := vm1.Exec("ip a | grep enp0s8 | grep inet | awk '{print $2}' | awk -F/ '{print $1}'")
+
+					// Wait the startup of the server
+					helpers.Sleep(1)
```

Comment:
That's likely to cause flakes. Could we instead move this server to a pod and check availability as we already do in other tests (see Wait helper functions in same file)?